cacti/tests/conftest.py

```diff
@@ -0,0 +1,58 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+import pytest
+from copy import deepcopy
+
+from datadog_checks.dev import docker_run
+import docker
+from datadog_checks.dev.subprocess import run_command
+from datadog_checks.cacti import CactiCheck
+
+from .common import (
+    HERE,
+    HOST,
+    INSTANCE_INTEGRATION
+)
+
+
+@pytest.fixture(scope="session")
+def dd_environment():
+    env = {'HOST': HOST}
+    with docker_run(
+        compose_file=os.path.join(HERE, "compose", "docker-compose.yaml"),
+        env_vars=env,
+    ):
+        yield INSTANCE_INTEGRATION
+
+
+@pytest.fixture
+def check():
```

Comment:
This really seems to me as a lack in our testing process. Cacti is not the only test which requires additional setup in the agent container. I also think of gunicorn that has to run of the very same container as the agent.
Anyway, this does the job nicely until we eventually change the testing process.